crates/trie/src/hashed_cursor/post_state.rs

```diff
@@ -179,36 +184,36 @@ pub struct HashedPostStateStorageCursor<'b, C> {
     cursor: C,
     /// The reference to the post state.
     post_state: &'b HashedPostStateSorted,
+    /// The current hashed account key.
+    hashed_address: B256,
     /// The post state index where the cursor is currently at.
     post_state_storage_index: usize,
-    /// The current hashed account key.
-    account: Option<B256>,
     /// The last slot that has been returned by the cursor.
     /// De facto, this is the cursor's position for the given account key.
     last_slot: Option<B256>,
 }
 
 impl<'b, C> HashedPostStateStorageCursor<'b, C> {
     /// Create new instance of [HashedPostStateStorageCursor].
```

Comment:
this should mention something like at the given hashed address

and should add a note that this does not check whether the storage is empty I think?